src/main/webapp/WEB-INF/views/index.jsp

```diff
@@ -16,6 +17,7 @@
     <tr><td><a href="${pageContext.request.contextPath}/drivers">Display All Drivers</a></td></tr>
     <tr><td><a href="${pageContext.request.contextPath}/cars">Display All Cars</a></td></tr>
     <tr><td><a href="${pageContext.request.contextPath}/manufacturers">Display All Manufacturers</a></td></tr>
+    <tr><td><a href="${pageContext.request.contextPath}/cars">Display my current Cars</a></td></tr>
```

Comment:
better /drivers/cars, because we already have /cars endpoint for Display all cars